tests/e2e/testsuites/e2e_utils.go

```diff
@@ -19,8 +19,27 @@ import "fmt"
 const (
 	DefaultVolumeName = "test-volume-1"
 	DefaultMountPath  = "/mnt/default-mount"
+
+	DefaultE2EIopsIoVolumes = "100"
```

Comment:
Drop `E2E` from the name (change it to `DefaultIopsIoVolumes`), you're in the e2e package so everything here is for the e2e tests, no need to specify.